src/components/scenes/MigrateWalletCalculateFeeScene.tsx

```diff
@@ -126,6 +127,8 @@ const MigrateWalletCalculateFeeComponent = (props: Props) => {
   // track of which makeSpends are successful so we can enable the slider. A single failure from any of a wallet's assets will cast them all as failures.
   useAsyncEffect(
     async () => {
+      if (!migrateWalletsSynced) return
```

Comment:
Optional: The total wait to sync (and then wait the extra five seconds for balances to hopefully work) is kind of long so maybe it's worth adding a toast like "fragment_transaction_list__synchronizing" as an extra heads up that stuff is working under the hood.

Reply:
Good call